index.html

```diff
@@ -75,6 +75,270 @@ <h1 class="headline-first-title">I'm Emmanuel Ikpe <br />
+                        required.</p>
+                    <ul class="work-langs">
+                        <li>
+                            <p>html</p>
+                        </li>
+                        <li>
+                            <p>css</p>
+                        </li>
+                        <li>
+                            <p>javaScript</p>
+                        </li>
+                    </ul>
+                    <button type="button" class="see-more">See Project</button>
+                </div>
+            </div>
+            <div class="work">
+                <img alt="work4" class="work-image" src="/assets/work4.png" />
+                <div>
+                    <div>
+                        <h2 class="work-title">Multi-Post Stories</h2>
+                        <ul class="work-details">
+                            <li>CANOPY</li>
+                            <li>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none"
+                                    xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </li>
+                            <li>Back End Dev</li>
+                            <li>
+                                <svg width="8" height="8" viewBox="0 0 8 8" fill="none"
+                                    xmlns="http://www.w3.org/2000/svg">
+                                    <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+                                </svg>
+                            </li>
+                            <li>2015</li>
+                        </ul>
+                    </div>
+                    <p class="work-des">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="work-langs">
+                        <li>
+                            <p>html</p>
+                        </li>
+                        <li>
+                            <p>css</p>
+                        </li>
+                        <li>
+                            <p>javaScript</p>
+                        </li>
+                    </ul>
+                    <button type="button" class="see-more">See Project</button>
+                </div>
+            </div>
+        </section>
+        <section class="about-section">
+            <div>
+                <h1 class="title">About
+                    Myself</h1>
+                <p class="details">Hello I’m a software developer! I can help you build a product , feature or website
+                    Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+                <button type="button" class="text-button">LET’S CONNECT</button>
```

Comment:
- [ ] This is a title and should not be wrapped in a button tag. Please use an `h1, h2, etc.`